mail-filter/mimedefang/files/mimedefang.init

```diff
@@ -0,0 +1,104 @@
+#!/sbin/openrc-run
+#
+# Generic start/stop script for MIMEDefang.  Should work on most
+# flavors of UNIX.
+
+RETVAL=0
+prog='mimedefang'
+PROGDIR=/usr/bin
+SPOOLDIR='/var/spool/MIMEDefang'
+PID="${SPOOLDIR}/${prog}.pid"
+MXPID="${SPOOLDIR}/${prog}-multiplexor.pid"
+LOCK="${SPOOLDIR}/${prog}.lock"
+MXLOCK="${SPOOLDIR}/${prog}-multiplexor.lock"
+
+# Make sure required vars are set
+SOCKET=${SOCKET:=${SPOOLDIR}/${prog}.sock}
```

Comment:
`a=${a:=x}` is redundant since `:=` assigns the value already. Just make it `: "${a:=x}"`.

Reply:
Changed the script in new commit.